src/client/sandbox/node/window.js

```diff
@@ -26,6 +26,7 @@ export default class WindowSandbox extends SandboxBase {
         this.messageSandbox = messageSandbox;
 
         this.UNCAUGHT_JS_ERROR_EVENT = 'hammerhead|event|uncaught-js-error';
+        this.IMAGE_FORCE_PROXY_SRC_FLAG = 'hammerhead|image|force-proxy-src-flag';
```

Comment:
Difficult to read in conditions.
I propose to rename this to `FORCE_PROXY_SRC_FOR_IMAGE_FLAG`